package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@open-craft/simple-theme",
+  "version": "1.0.0",
```

Comment:
I believe 0.1.0 would be [more suitable](https://semver.org/#how-do-i-know-when-to-release-100) here.